pkg/utils/utils.go

```diff
@@ -921,6 +927,22 @@ func RunCommand(command string, args ...string) (string, error) {
 	return stdout.String(), err
 }
 
+func HashConfigMap(cm *corev1.ConfigMap) string {
```

Comment:
add UT to make sure this behaves as expected ?

Reply:
Some kind of UT like this
```
	configData := make(map[string]string)
	jsonData1 := []byte(`{
	  "resourceList": [
	    {
	      "resourceName": "mgmtvf",
	      "selectors": {
	        "pfNames": [
	          "ens7f0#0-0"
	        ],
	        "IsRdma": false,
	        "NeedVhostNet": false
	      },
	      "SelectorObj": null
	    },
	    {
	      "resourceName": "mlxnics",
	      "selectors": {
	        "pfNames": [
	          "ens7f0#1-11"
	        ],
	        "IsRdma": false,
	        "NeedVhostNet": false
	      },
	      "SelectorObj": null
	    }
	  ]
	}`)
	jsonData2 := []byte(`{
	  "resourceList": [
	    {
	      "resourceName": "mgmtvf",
	      "selectors": {
	        "pfNames": [
	          "ens8f0#0-0"
	        ],
	        "IsRdma": false,
	        "NeedVhostNet": false
	      },
	      "SelectorObj": null
	    },
	    {
	      "resourceName": "mlxnics",
	      "selectors": {
	        "pfNames": [
	          "ens9f0#1-11"
	        ],
	        "IsRdma": false,
	        "NeedVhostNet": false
	      },
	      "SelectorObj": null
	    }
	  ]
	}`)

	configData["worker-225"] = string(jsonData1)
	configData["worker-226"] = string(jsonData2)

	cm := &corev1.ConfigMap{
		TypeMeta: metav1.TypeMeta{
			APIVersion: "v1",
			Kind:       "ConfigMap",
		},
		ObjectMeta: metav1.ObjectMeta{
			Name:      "my-config-map",
			Namespace: "my-namespace",
		},
		Data: configData,
	}

	hash := HashConfigMap(cm)
```

Reply:
Unit tests looks good Balazs!